src/main/java/org/example/Main.java

```diff
@@ -1,11 +1,43 @@
 package org.example;
 
-import org.example.entity.*;
 
-public class Main {
+import org.apache.logging.log4j.*;
+
 
+import org.example.entity.Agent;
+import org.example.entity.Property;
+import org.example.entity.RentData;
+import org.example.entity.Tenant;
+import org.example.repository.CentralRepository;
 
+public class Main {
+
+private static final Logger logger = LogManager.getLogger(Main.class);
     public static void main(String[] args) {
 
+
+
+        CentralRepository repository = CentralRepository.getINSTANCE();
+        repository.generateSomeData();
```

Comment:
generateRandomData() or generateTestData() -> some data may just confuse a little be, try to be more precise.